app/models/facility_group.rb

```diff
@@ -24,8 +24,26 @@ class FacilityGroup < ApplicationRecord
   friendly_id :name, use: :slugged
 
   auto_strip_attributes :name, squish: true, upcase_first: true
+  attribute :enable_diabetes_management
+
+  before_save :enable_diabetes_management!, if: -> { enable_diabetes_management }
+  before_save :disable_diabetes_management!, if: -> { !enable_diabetes_management && diabetes_enabled? }
```

Comment:
Putting this in the model layer may be risky. What happens when a facility group that is fully DM-enabled, is, say, edited through the console or a data migration? Would `facility_group.touch` disable DM in all of its facilities?

Reply:
That is correct, doing this on console for example:
```
facility_group.update(name: "name")
```
will set all facilities to `false`. Replacing this with a regular method 😕 , easier to wrap heads around.

Reply:
Neat. Yup, a couple extra lines in the controller should be ok 👍

Reply:
This is done:

![image](https://user-images.githubusercontent.com/50663/85138995-6720dc00-b261-11ea-8b49-d8f8ef5262f3.png)